README.md

```diff
@@ -140,26 +140,26 @@ A reducer that keeps track of Router state.
 
 A component that renders a React Router app using router state from a Redux store.
 
-### `pushState(state, pathname, query)`
+### `push(state, pathname, query)`
 
-An action creator for `history.pushState()`. (https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)
+An action creator for `history.push()`. (https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)
```

Comment:
We should probably reference https://github.com/mjackson/history/blob/83f1a6382bd03b04d334958f44be6c9362b920a5/docs/GettingStarted.md#navigation or something from the history docs instead, since its an action creator for that API (which then proxies to the browser API)